install_config/install/host_preparation.adoc

```diff
@@ -245,7 +245,7 @@ Installation] method, you can easily script a complete installation from a
 kickstart or cloud-init setup, change the default configuration file:
 +
 ----
-# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16"' \
+# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16" --signature-verification=false'
```

Comment:
1.  We shouldn't changed the default log driver (default value is `json-file`)

2.  The options including "--signature-verification=false" should be quoted together

This command makes sense to me: 
` sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --log-driver=journald --insecure-registry 172.30.0.0/16 --signature-verification=false"' /etc/sysconfig/docker`

To keep backward and forward compatibility, I suggest that we just need to document to add extra parameter `--insecure-registry 172.30.0.0/16` for `OPTIONS` instead of posting a shell command.